src/config/dark-sites.config

```diff
@@ -502,6 +502,10 @@ humblegames.com
 hyper.is
 hyperbeam.dev
 i3wm.org
+iaas.store
+*.iaas.store
```

Comment:
Is this site has some subpages that you need this alias?
If yes alias must be added in the beginning of the list as it must be alphabetical ordered.

Reply:
there are several pages (dashboard., admin. , auth.), there will most likely be more of them and they all support a dark theme. I wrote * to apply them all, is that okay?

Reply:
You can add *. but they must be moved into the beginning of the list or tests will reject them.
Summarizing: they can not be added just under iaas.store

P.S. Same goes to *.iaasdev.ru